api/models/roomDao.js

```diff
@@ -0,0 +1,23 @@
+import { dataSource } from './dataSource.js';
+
+const roomsByHost = async (userId) => {
+  try {
+    const rooms = await dataSource.query(
+      `
+      SELECT id, restaurant_id, host_id, image, date, time, max_num, age_id, gender_id from rooms
+      WHERE host_id = ?;
```

Comment:
1. 가져오는 column들 잘 볼 수 있게 아래와 같이 정렬해 주세요.
2. 주고 받는 데이터 형식이 snake_case 인가요? 그렇지 않다면 AS를 이요해서 camel_case로 바꿔 주세요.
```suggestion
      SELECT 
        id, 
        restaurant_id AS restaurantId,
        host_id, 
        image, date, 
        time, 
        max_num, 
        age_id, 
        gender_id 
      FROM rooms
      WHERE host_id = ?;
```